modules/inflowwind/src/InflowWind.txt

```diff
@@ -28,34 +23,11 @@ param    ^                       -                 IntKi             HAWC_WindNu
 param    ^                       -                 IntKi             User_WindNumber      -     6     -     "User defined wind."                                                       -
 param    ^                       -                 IntKi             BladedFF_Shr_WindNumber  - 7     -     "Native Bladed binary full-field file."      -
 param    ^                       -                 IntKi             FDext_WindNumber     -     8     -     "4D wind from external souce (i.e., FAST.Farm)."                                                       -
-param    ^                       -                 IntKi             Highest_WindNumber   -     8     -     "Highest wind number supported."                                           -
+param    ^                       -                 IntKi             Point_WindNumber     -     9     -     "4D wind from external souce (i.e., FAST.Farm)."                                                       -
```

Comment:
Do we need the `Point_WindNumber`?  I think FAST.Farm might be able to use the `FDext_WindNumber`.

Reply:
I didn't update the comment. `Point_WindNumber` is for the wind from `ExtInflow` which is an array of wind components corresponding to positions.

Reply:
The comment was confusing me.  Maybe we change the comment on `Point_WindNumber` to `Wind vectors at a list of point locations (used by ExtInflow)` or similar?